src/shouldProvide.ts

```diff
@@ -50,8 +51,15 @@ function startsWithADot(textCurrentLine: string, position: number) {
         && textWithinString[0] === '.';
 }
 
+function startsWithAtPath(textCurrentLine: string, position: number) {
```

Comment:
```suggestion
function startsWithAnAtSymbol(textCurrentLine: string, position: number) {
```
Hey, you changed the name as I suggested, but you left the function name itself unchanged.
You should update that too.
Have a nice day 😄

Reply:
Sorry about that. I'm outside now so I used github's "Add suggestion to batch" function to merge your change. I'll fix it on Monday.

Reply:
done